bike_rent/models/product.py

```diff
@@ -5,6 +5,7 @@ class ProductTemplate(models.Model):
     _inherit = 'product.template'
 
     is_bike = fields.Boolean(string="It's a Bike")
+    bike_shop = fields.Boolean(string='Available in Shop')
```

Comment:
why add another parameter to the product table, you have enough already to filter out rents, that are `Available in Shop`? `sale_ok` field in meant exactly for that purpose.